pkg/nsx/services/mediator/mediator.go

```diff
@@ -14,3 +23,25 @@ type ServiceMediator struct {
 	*securitypolicy.SecurityPolicyService
 	*vpc.VPCService
 }
+
+// GetOrgProjectVPC is a common method, extracting the org and project string from vpc path of the VPC model.
+// VPC path looks like "/orgs/default/projects/project-1/vpcs/vpc-1",
+// Since other modules only know namespace, this is the only entry point to get org and project.
+// Currently, we only support one vpc per namespace, but we may support multiple vpcs per namespace in the future,
+// so we return a slice of OrgProject.
+func (serviceMediator *ServiceMediator) GetOrgProjectVPC(ns string) []common.OrgProjectVPC {
+	var orgProjectVPC []common.OrgProjectVPC
+	vpcs := serviceMediator.GetVPCsByNamespace(ns) // Transparently call the VPCService.GetVPCsByNamespace method
+	for _, v := range vpcs {
+		matches := reExp.FindStringSubmatch(*v.Path)
+		if len(matches) == 4 {
+			org := matches[1]
+			project := matches[2]
+			vpc_ := matches[3]
```

Comment:
why add "_" suffix for "vpc_" variable?

Reply:
IDE reminds 
<img width="560" alt="image" src="https://user-images.githubusercontent.com/4385934/231635825-77f18e79-7f20-4030-a96b-5e06929fe4b6.png">
so add this trick.